examples/schelling/model.py

```diff
@@ -6,24 +6,21 @@ class SchellingAgent(mesa.Agent):
     Schelling segregation agent
     """
 
-    def __init__(self, model, agent_type):
+    def __init__(self, model: mesa.Model, agent_type: int) -> None:
         """
         Create a new Schelling agent.
```

Comment:
This docstring is self explanatory for anyone knowing what `__init__` does, and should be considered a superfluous documentation.

Reply:
I have read several articles about superfluous documentation, but couldn't find a good reference that explains it. Maybe this [SO thread](https://stackoverflow.com/questions/33019955/should-you-always-document-functions-even-if-redundant-specifically-python) does it.